doc/source/tutorial/stats/continuous_t.rst

```diff
@@ -37,12 +37,11 @@ As :math:`\nu\rightarrow\infty,` this distribution approaches the standard norma
 
 .. math::
 
-     h\left[X\right]=\frac{1}{4}\log\left(\frac{\pi c\Gamma^{2}\left(\frac{c}{2}\right)}{\Gamma^{2}\left(\frac{c+1}{2}\right)}\right)-\frac{\left(c+1\right)}{4}\left[\Psi\left(\frac{c}{2}\right)-cZ\left(c\right)+\pi\tan\left(\frac{\pi c}{2}\right)+\gamma+2\log2\right]
+     h\left[X\right]=\frac{\nu+1}{2} \left[\psi \left(\frac{1+\nu}{2} \right) -\psi \left(\frac{\nu}{2} \right) \right] + \ln \left[ \sqrt{\nu} B \left( \frac{\nu}{2}, \frac{1}{2} \right) \right]
 
```

Comment:
```suggestion

where :math:`\psi(x)` is the digamma function and :math:`B(x, y)` is the
beta function.

```
Add references for those function if you think they are needed, but I'm OK with just the suggested addition.

Reply:
OK. I didn't include them because they are defined for all the stats tutorials [here](https://23779-1460385-gh.circle-artifacts.com/0/html-scipyorg/tutorial/stats/continuous.html#references). But committing.